src/App.svelte

```diff
@@ -36,6 +38,11 @@
     <div class="body">
       <div
         class="text-center container mx-auto px-4 sm:px-6 lg:px-8 flex flex-col align-center">
+        <div class='container'> <Excursion/> 
+
+      </div>
+       
+
```

Comment:
what happened here?

Reply:
IDK dude but I was hoping my linter would auto fix that stuff. It didn't tho !!